CSS/modules.css

```diff
@@ -0,0 +1,120 @@
+/*******  CSS/MODULES ********/
+
+/*******  HEADER/ NAV  ********/
+
+.header__div__h1 {
+  font-family: 'Yellowtail', cursive;
+  font-size: 2.2em;
+  font-weight:100;
+}
+.header__div__p {
+  font-family: 'Abel', sans-serif;
+  color: gray;
+  font-size: .7em;
+  margin-top: -5px;
+}
+
+  nav a:nth-child(6) {
+    padding-right: .5em;
+  }
+
+/*******  MAIN  ********/
+
+main {
+  font-family: 'Droid Serif', serif;
+  color: gray;
+}
+  .main__div {
+    border-top: solid lightgray;
+    border-top-width: 15px;
+    /*  border-image: url(../dash-border.png);
+    border-image-repeat: repeat;*/
+  }
+    .main__div__h1 {
+      font-family: 'Oswald', sans-serif;
+      font-size: 2em;
+      color:black;
+    }
+    .main__div__h3 {
+      font-size: .6em;
+    }
+
+  .print {
+    font-size: .7em;
+  }
+
+  .main__img{
+    max-width: 100%;
```

Comment:
Make this just "width: 100%" for images since they inherit width from their parent element.